R/input.R

```diff
@@ -212,7 +219,7 @@ numeric_input <- function(input_id, label, value, min = NA, max = NA, step = NA,
 #' @param ... Other parameters passed to \code{\link{numeric_input}} like \code{type} or \code{icon}.
 #' @rdname numeric_input
 #' @export
-numericInput <- function(inputId, label, value,
+numericInput <- function(inputId, label, value = NULL,
```

Comment:
I would suggest adding `placeholder` as a named argument and adding the information about the requirement (needs value or placeholder) to the documentation.

Reply:
Done